system-addon/lib/ManualMigration.jsm

```diff
@@ -51,8 +55,9 @@ this.ManualMigration = class ManualMigration {
    * @param {bool} alreadyExpired Pref flag that is false for the first 3 active days,
    *                              time in which we display the migration message to the user.
    */
-  expireIfNecessary(alreadyExpired) {
-    if (!alreadyExpired && this.isMigrationMessageExpired()) {
+  async expireIfNecessary(alreadyExpired) {
+    const isMigrationExpired = await this.isMigrationMessageExpired();
+    if (!alreadyExpired && isMigrationExpired) {
```

Comment:
adding `await` in the same original line is well within 80 characters on the line:
`    if (!alreadyExpired && await this.isMigrationMessageExpired()) {`